packages/views/utilities/externalLinks.js

```diff
@@ -0,0 +1,32 @@
+import history from '@common/utilities/history';
+
+const sendRegex = /^\/(wallet|wallet\/send|main\/transactions\/send)$/;
```

Comment:
please read this article and avoid regex as you can!
https://siderite.dev/blog/why-people-dont-use-regular-expressions/

Reply:
@soroushm I've read the article and while it makes sense to avoid regex in a lot of cases especially complex regex, this seems like such a simple case that it can be used. However I'm open to other opinions and how best to proceed

Reply:
@ikem-legend I'll happy to share with me by tomorrow